chore: format code blocks (#18799)
newrelic/docs-website: src/content/docs/nrql/nrql-syntax-clauses-functions.mdx

          <tr>
            <td>
              `valueC[-1:]
            </td>

            <td>

>               Returns the last _n_ list values, where _n_ is a negative number before the colon(i.e `[-n:]`). In this case the list value at index `2` will be returned.

Reviewer: ```suggestion
              Returns the last **n** list values, where **n** is a negative number before the colon(for example, `[-n:]`). In this case the list value at index `2` will be returned.
```

Author: Bold instead of italics is fine with me, but I made the same change of `n` -> `_n_` in another part of the doc as well.

Is this related to the run failing? before the changed I added today this PR was building fine but I see no issues with my changes from this morning

Author: I changed both to use `**`
